src/resources/2006/CommaChameleon.ts

```diff
@@ -0,0 +1,31 @@
+import { $familiar } from "../../template-string.js";
+import { have as _have } from "../../lib.js";
```

Comment:
```suggestion
import { have as have_ } from "../../lib.js";
```

putting a `_` in front is for when something is intended to be private; putting a `_` in the back is used for name collisions

Reply:
Odd. I copied from the Bander code.